x-pack/plugins/apm/public/components/app/TraceOverview/view.tsx

```diff
@@ -23,7 +24,7 @@ export function TraceOverview(props: Props) {
       <EuiSpacer />
       <TraceListRequest
         urlParams={urlParams}
-        render={({ data }: { data: object[] }) => (
+        render={({ data }: { data: ImpactItem[] }) => (
```

Comment:
Doesn't `TraceListRequest` return a list of trace items, and not impact items?

Reply:
Btw. it would be cool if we could add an interface to the node endpoints, and re-use that here! :)